### ISSUE TITLE
Automatic braille display selection fails initially if remote connected

### ISSUE THREAD

**Issue by XLTechie** [NVDARemote/NVDARemote#299]
While supporting a speech and braille user via remote, we noticed that any time we restarted NVDA with my remote session connected, his braille display would remain blank.
To cause it to initialize, it was necessary to open NVDA braille display selection, and then press OK to close it again. (Desktop key sequence: NVDA+Ctrl+A, Enter)

This situation has actually been around for a while, but I had never taken the time to diagnose it.

The braille display is a Hims Q-Braille, although it may have happened with an Orbit 40 in the past.

I think the important factor is that the NVDA braille driver is set to automatic. Both before and after it seems to detect the correct display, but on NVDA startup for some reason it just doesn't receive any output.

This only happens with the Remote add-on installed, currently version 2.5, and NVDA 2022.2 on Windows 10 64 21H1, although ealier versions of everything have had at least some version of the same problem, although it may have been intermittent before.

[log of no automatic braille with NVDA Remote.txt](https://github.com/NVDARemote/NVDARemote/files/9233263/log.of.no.automatic.braille.with.NVDA.Remote.txt)

CC @tspivey
